src/components/HighlightedLink/HighlightedLink.module.css

```diff
@@ -0,0 +1,15 @@
+.link {
+  color: hsla(0, 0%, 0%, 1);
+  font-weight: 600;
+  background: linear-gradient(
+    to top,
+    rgba(255, 215, 0, 0.4) 35%,
+    transparent 35%
+  );
+  text-decoration: none;
```

Comment:
you could pull this out into a global rule for a tags

`a {
  text-decoration: none;
}`